docs/names-and-ids.md

```diff
@@ -0,0 +1,43 @@
+# Names and IDs
+
+version .1
+
+There are a number of names and IDs used in THIEF.  This table describes some of them:
+
+| Identifier | format | Meaning |
+| - | - | - |
+| servicePool | string | Name of pool that a service app belongs to. |
+| requestedServicePool | string | Name of pool that a device app would like to pair with. |
+| runId | guid | Guid representing a running executable. |
+| serviceAppRunId | guid | Guid representing a running _service_ executable. |
+| sessionId | guid | Guid representing the pairing between a device app and a service app. |
+| pingbackId | guid | Guid used to represent the ACK of a thief operation. |
+
+## servicePool notes
```

Comment:
`servicePool` should either be capitalized or, if it is a variable, should be wrapped in backticks